tests/Captioning/Format/WebvttFileTest.php

```diff
@@ -4,6 +4,146 @@
 
 class WebvttFileTest extends \PHPUnit_Framework_TestCase
```

Comment:
Additional unit tests. I get some examples from specification and test parse method. After that I also tested exeptions what parse method can throws throuh parsing process.